Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Nodes.php

```diff
@@ -194,4 +197,20 @@ public function nextAll(Node $referenceNode): self
 
         return new self(array_slice($this->nodes, $referenceNodeIndex + 1));
     }
+
+    /**
+     * @param \Closure(Node $node): mixed $callback
+     * @return array<mixed>
+     */
+    public function map(\Closure $callback): array
+    {
+        return array_map($callback, $this->nodes);
+    }
+
+    public function mapToNodeAggregateIds(): NodeAggregateIds
```

Comment:
```suggestion
    public function toNodeAggregateIds(): NodeAggregateIds
```

otherwise confusing, imho

Reply:
Fine by me